.github/workflows/deploy-env.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Environment
+
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to'
+        type: choice
+        required: true
+        default: 'ops-preview-1'
+        options:
+        - ALL
+        - LIVE
```

Comment:
If, say, `live-preprod` is split between a few regions, should we also have a `LIVE-PREPROD` group? Or will they always be rolled out one by one?

Reply:
Yes. Since the envs are in different clusters that act as backup for each other, it is a good idea to roll them out one by one; a disaster-recovery, blue-green deployment style, if you will.